### DISCUSSION TITLE
Quick Start - Can i access Language, Phone number and SMS gateway features?

### DISCUSSION THREAD

**Discussion by ankitbhansali** [egovernments/Digit-Core#122] [DIGIT Platform]
Hello, i want to see if i can change the language, phone number code to specific country and also integrate with SMS gateway. Is it possible to use these features in quick start setup?

Also one more thing, can we deploy DIGIT on our own VM or server?
Thank you.

**Comment by talele08** [comment]
Yes you can deploy DIGIT on VM or server. Basically you need to first setup kubernetes on your server and then you can run the DIGIT services on that cluster. Once you deploy this services you can do the customisation
